start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-
-. ../logger-env/bin/activate
-nohup python music_logger.py &
+/usr/bin/cd /home/logger/music_logger &&
+/bin/sh /home/logger/logger-env/bin/activate &&
+/usr/bin/nohup /home/logger/logger-env/bin/python /home/logger/music_logger/music_logger.py &
```

Comment:
This script would better be suited as a systemd unit, or at least a supervisord config. Those options are more robust (provide for easy restarting and status checking, and automated restart on failure) and systemd units automatically capture STDOUT and STDERR to journal/syslog, so you get nice logging for free.